CIP-XXXX/README.md

```diff
@@ -0,0 +1,1601 @@
+---
+CIP: ?
+Title: Full-data wallet connector
+Category: Tools
+Status: Proposed
+Authors:
+  - Giovanni Garufi <giovanni@mlabs.city>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/957
+Created: 2024-12-13
+License: CC-BY-4.0
+Version-Connection-API: 0.0.0
+Version-CIP-30-Extension: 0.0.0
```

Comment:
```suggestion
```
@nazrhom - at this time, we have no such headers in the CIP front matter.  Adding them could be ignored by some parsers (not all of which are YAML parsers: some have just autistically cut up the text) and generate errors on others.

Until we get feedback from the owners of https://github.com/cardano-foundation/cf-cip-frontend we will have to keep these fields to the ones defined here: https://github.com/cardano-foundation/CIPs/blob/master/CIP-0001/README.md#header-preamble

Reply:
p.s. I did also meant to say that, in my opinion, what you've written in the (appreciated) Versioning section describes the version state & progression adequately.

Reply:
Thanks @rphair I have created a separate table in the versioning section to keep track of those

Reply:
yes, that's exactly what we are looking for 👍 though I might suggest adding a header line to that table (or removing the line with the empty column headers)